src/serialization/KiotaSerializer.Deserialization.cs

```diff
@@ -31,19 +34,21 @@ private static Stream GetStreamFromString(string source)
     {
         var stream = new MemoryStream();
```

Comment:
this method should also be marked obsolete to ensure we don't end up using it somewhere else by mistake and to ensure all consumers are also marked as obsolete

Reply:
I've update changelog (is it ok like that?) but didn't update the version number yet. Should it be 1.8.5 or (probably) 1.9.0 since it has quite some changes?
"_I think we should preview/validate the changes for the JSON serialization library before we merge this one in, just in case we missed something, so we can validate the design. Thoughts?_" That'd be certainly the best way, do you have any ideas how to?

Reply:
1. Should be 1.9.0 given that we are adding new apis here.
2. You can pull in https://github.com/microsoft/kiota-serialization-json-dotnet and replace the abstractions dependency in the project with this branch to build and test locally. (You can checkout line 14 and 66 in [this commit](https://github.com/microsoft/kiota-serialization-json-dotnet/commit/d152ee8667942c6f176a0efdbab3d7141568866c) if it helps.)

Reply:
Thanks! Can you stand up the json PR as a draft please?

Reply:
Done